scripts/components/LviHomeCategories.ts

```diff
@@ -0,0 +1,49 @@
+import { getCombinedStyle } from '@smartface/extension-utils/lib/getCombinedStyle';
+import LviHomeCategoriesDesign from 'generated/my-components/LviHomeCategories';
+import GviHomeCategoryItem from './GviHomeCategoryItem';
+const { height } = getCombinedStyle('.lviHomeCategories').height;
+export default class LviHomeCategories extends LviHomeCategoriesDesign {
+    pageName?: string | undefined;
+    private __items: any[] = [];
+    constructor(props?: any, pageName?: string) {
+        // Initalizes super class for this scope
+        super(props);
+        this.pageName = pageName;
+    }
+    static getHeight(): number {
+        return height;
+    }
+    get items(): any[] {
+        return this.__items;
+    }
+    set items(value: any[]) {
+        this.__items = value;
+        this.initGridView();
```

Comment:
we shouldn't be setting onItemBind everytime items get set. We need a initial function that sets binding probably inside constructor.